BladeportBinaryTreeManager.Business.Tests/UserBLTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿using BladeportBinaryTreeManager.Contracts;
+using BladeportBinaryTreeManager.DTO;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace BladeportBinaryTreeManager.Business.Tests
+{
+	[TestFixture]
+	public class UserBLTests
+	{
+		private List<UserDTO> userList;
+		private List<UserDTO> emptyUserList;
+		private Mock<IUserBL> _userBl;
+
+		[SetUp]
+		public void Setup()
+        {
+			userList = new List<UserDTO>()
+			{
+				new UserDTO { UserId = 1, UserName="johndoe", FirstName = "Joe", LastName = "Doe", JoinDate = new System.DateTime() },
+				new UserDTO { UserId = 1, UserName="wandajones", FirstName = "Wanda", LastName = "Jones", JoinDate = new System.DateTime() },
+				new UserDTO { UserId = 1, UserName="sidepocket88", FirstName = "Sid", LastName = "Myers", JoinDate = new System.DateTime() },
+				new UserDTO { UserId = 1, UserName="mercedes123", FirstName = "Mercedes", LastName = "Benz", JoinDate = new System.DateTime() },
+				new UserDTO { UserId = 1, UserName="robford", FirstName = "Robert", LastName = "Ford", JoinDate = new System.DateTime() }
+			};
+
+			emptyUserList = new List<UserDTO>();
+		}
+
+		[Test]
+		public void GetUserList_NotEmpty_Test()
+        {
+			// Arrange
+			_userBl = new Mock<IUserBL>();
+			_userBl.Setup(users => users.GetUserList()).Returns(() => userList);
+
+			// Act
+			var result = _userBl.Object.GetUserList();
+
+			// Assert
+			Assert.IsTrue(result.Count > 0);
```

Comment:
Import the FluentAssertion Nuget package and use it to Assert Unit Tests (include existing tests)

Reply:
In FluentAssertion, this can now be asserted => result.Count().Should().BeGreaterThan(0);